tailwind.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './ui/**/*.{js,jsx,ts,tsx}',
+    './node_modules/@metamask/design-system-react/**/*.{cjs,mjs}',
+  ],
+  presets: [
+    // eslint-disable-next-line node/global-require
+    require('@metamask/design-system-tailwind-preset'),
+  ],
+  theme: {
+    colors: {}, // This removes all default Tailwind colors
+    extend: {},
+  },
+  plugins: [],
+};
```

Comment:
This is a Tailwind CSS configuration file that:

1. Tells Tailwind to scan UI files and MetaMask's design system files for CSS classes
2. Uses MetaMask's design system preset for styling
3. Removes default Tailwind colors (we only need the colors provided by the design system preset)

Reply:
Why do some build systems get away with calling `tailwindcss()` with no args, but the gulp system needs the config file?

Reply:
Great catch! I was under the impression it was needed for the gulp build but I just tested it and it works without having to pass the config file. Updated here https://github.com/MetaMask/metamask-extension/pull/30170/commits/2581835160affbb1aa429753305b32b4ecd3d585

Reply:
Just kidding it is needed. I should have tried building as well as `yarn start` when I was testing. Reverted here 🤦 https://github.com/MetaMask/metamask-extension/pull/30170/commits/2fb8ae7a77f1e922ff7814018549a326173f08de

<img width="1098" alt="Screenshot 2025-07-09 at 3 18 57 PM" src="https://github.com/user-attachments/assets/b44cb36a-794f-4c62-aa47-2495e4e36f19" />